tests/DCPS/BuiltInTopic/main.cpp

```diff
@@ -516,8 +516,8 @@ ACE_TMAIN (int argc, ACE_TCHAR *argv[])
       bit_subscriber = participant->get_builtin_subscriber () ;
       TEST_CHECK (! CORBA::is_nil (bit_subscriber.in ()));
 
-      test_bit_participant ();
-      test_bit_topic ();
+      //test_bit_participant ();
+      //test_bit_topic ();
```

Comment:
I think you meant to uncomment this out at some point.

Reply:
I reverted the change. Thanks.